src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext.md

```diff
@@ -10,17 +10,18 @@ The [browser module API](/javascript-api/k6-experimental/browser#browser-module-
-| [BrowserContext.newPage()](/javascript-api/k6-experimental/browser/browsercontext/newpage)                                                                  | Uses the `BrowserContext` to create a new [Page](/javascript-api/k6-experimental/browser/page/) and returns it.        |
-| [BrowserContext.pages()](/javascript-api/k6-experimental/browser/browsercontext/pages) <BWIPT id="444"/>                                                    | Returns a list of [page](/javascript-api/k6-experimental/browser/page/)s that belongs to the `BrowserContext`.          |
-| [BrowserContext.setDefaultNavigationTimeout(timeout)](/javascript-api/k6-experimental/browser/browsercontext/setdefaultnavigationtimeout) <BWIPT id="445"/> | Sets the default navigation timeout in milliseconds.                                                       |
-| [BrowserContext.setDefaultTimeout(timeout)](/javascript-api/k6-experimental/browser/browsercontext/setdefaulttimeout) <BWIPT id="456"/>                     | Sets the default maximum timeout for all methods accepting a timeout option in milliseconds.               |
-| [BrowserContext.setGeolocation(geolocation)](/javascript-api/k6-experimental/browser/browsercontext/setgeolocation) <BWIPT id="435"/>                       | Sets the `BrowserContext`'s geolocation.                                                                   |
-| [BrowserContext.setOffline(offline)](/javascript-api/k6-experimental/browser/browsercontext/setoffline)                                                     | Toggles the `BrowserContext`'s connectivity on/off.                                                        |
-| [BrowserContext.waitForEvent(event[, optionsOrPredicate])](/javascript-api/k6-experimental/browser/browsercontext/waitforevent) <BWIPT id="447"/>           | Waits for the event to fire and passes its value into the predicate function.                              |
+| Method                                                                                                                                                      | Description                                                                                                     |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| [BrowserContext.addCookies()](/javascript-api/k6-experimental/browser/browsercontext/addcookies/)                                                           | Adds cookies into the `BrowserContext`.                                                                         |
```

Comment:
```suggestion
| [BrowserContext.addCookies()](/javascript-api/k6-experimental/browser/browsercontext/addcookies/)                                                           | Adds [cookies](/javascript-api/k6-experimental/browser/browsercontext/cookie) into the `BrowserContext`.                                                                         |
```

WDYT?

Reply:
Thanks for the eagle eyes you have: https://github.com/grafana/k6-docs/pull/1328/commits/c942397ab6b5bf67fd9feef95706f2cad2be64f7